src/execution_plan/ops/op_update.c

```diff
@@ -149,25 +161,64 @@ static Record _handoff(OpUpdate *op) {
 	return NULL;
 }
 
-OpBase *NewUpdateOp(const ExecutionPlan *plan, EntityUpdateEvalCtx *update_exps) {
+static void _groupUpdateExps(OpUpdate *op, EntityUpdateEvalCtx *update_ctxs) {
+	// Sort update contexts by unique Record ID.
+#define islt(a,b) (a->record_idx < b->record_idx)
```

Comment:
```suggestion
    #define islt(a,b) (a->record_idx < b->record_idx)
```

Reply:
My autoformatter forces this, and a bit of research indicates that no whitespace preceding preprocessor directives is standard form:
https://stackoverflow.com/a/34624201/1129030
If you guys feel strongly about this, I'll try to find a fix!